### PR TITLE
Update starter to the latest

### PR DESCRIPTION
**Pull request by IzumiSy**
Update starter for more information

### CODE REVIEW

File: src/app/page.tsx
Comment (IzumiSy) [inline src/app/page.tsx:1]: Updated initial page to use `@tailor-platform/design-systems` and `styled-system` generated by PandaCSS

File: .eslintrc.js
Comment (IzumiSy) [inline .eslintrc.js:1]: Use @tailor-platform/dev-config

File: tailwind.config.js
Comment (IzumiSy) [inline tailwind.config.js:1]: We don't use tailwindCSS

File: tsconfig.json
Comment (IzumiSy) [inline tsconfig.json:1]: Use @tailor-platform/dev-config

File: panda.config.ts
Comment (IzumiSy) [inline panda.config.ts:1]: Minimum configuration to use pandaCSS with our design-systems

**Review by yukito5669 — APPROVED** [review]

**Review by fuzeyerr — APPROVED** [review]

**Pull request merged**